### DISCUSSION TITLE
Intent 7 analysis

### DISCUSSION THREAD

**Discussion by Kevsy** [camaraproject/EdgeCloud#178] [General]
### Developer intents: Provisioning intents

| Intent 7 | 5SED |5MEE |EXRA|EXRC| EXCA | EXSM|  GMEC |
| -------- |------| ----|----| ---|------|-----|-------|
| 7. “I can ask the operator to store artifacts e.g., container images or VM images and manifests describing required resources, Helm charts etc”  | No           |No    | ?          | ?     | ?            | ?     | Yes   |

| GMEC        | Details       |Comments|
| --------------|-------------| -----------|
| (to complete) | (to complete) | (to complete) |

(others to complete as appropriate)

**Comment by gunjald** [comment]
Yes, the application management APIs proposes the application provider to submit application component images e.g. containers, QCOW2 etc, application descriptors e.g. Helm charts
